### DISCUSSION TITLE
字体大小设置不生效

### DISCUSSION THREAD

**Discussion by zhbfy** [stone-zeng/fduthesis#267] [Q&A] [answered]
您好，用的overleaf版本，做了如下设置，但是字体仍然是五号

```
style = {
    cjk-font = fandol,
    font-size = -4,
    footnote-style = xits,
    bib-backend = bibtex,
    bib-resource = {fduthesis-template.bib},
  }
```

然后有下面这样一个警告

```
Package relsize Warning: Failed to get list of defined font sizes. Size scaling will attempt arbitrary sizes.
```

想咨询一下您，大概是什么原因

**Comment by note286** [comment, marked as answer]
我这边直接打开 [fduthesis - LaTeX thesis template for Fudan University - Overleaf, Online LaTeX Editor](https://www.overleaf.com/latex/templates/fduthesis-latex-thesis-template-for-fudan-university/svtdhhstkmkt)。只更改`font-size = 5,`为`font-size = -4,` 正文字号是有变化的。

**Reply by zhbfy** [reply]
我也试了，确实空的就没问题，加了我自己的正文就有问题了

**Reply by note286** [reply]
那你检查一下你引入了什么命令，你不方便提供测试用例的话是没法测试的。

**Reply by stone-zeng** [reply]
是不是因为调用了 `relsize` 包导致的？

**Reply by zhbfy** [reply]
我弄好了，谢谢两位
